_posts/2023-02-17-7th-workshop-announcement.md

```diff
@@ -0,0 +1,140 @@
+- Hiking boots
+- Suitable clothing
+- Rain pans 
+- Waterproof jacket 
+- Hat
+- Snacks/Food.
+- Small backpack
+- Water bottle or thermos. 
+- Sunglasses or sunscreen (in case the Scotish weather is kind)
+- Make sure you have appropriate health insurance.
+
+<b> Drinks and food </b>
+
+- Breakfast, Coffe/Tea breaks and Lunch will be provided and are included in the fee of the workshop. 
+- We will have at least one social dinner also included in the workshop fee. (And probably a second social activity in the middle of the week)
+- Note that we will not provide all Dinner meals for the whole week but we will point out good spots to find dinner near the University accommodation and ECCI.
+- As Scotland's capital city, Edinburgh is full of food options as is known for its unique and wonderful nightlife. Please check out the [Visiting Edinburgh](https://edinburgh.org/) page so you can make the most of your stay in the city.
+
+### COVID protocol
+**I guess we need to decide on this as it does not seem to be much of an issue? At least in the UK all rules have been relax**
+
+Recent covid outbreaks at conferences and workshops have shown how disruptive it can be, 
+with many people being sick and stuck abroad. This is something we really want to prevent. 
+Even more because the research station is both remote (with no medical facilities in its 
+surroundings) and will close right after our stay. Taking this in consideration, combined with 
+that we will be living close together, we have decided on the following protocol:
+
+- Everybody needs to bring at least 5 covid rapid-tests. Everybody should take a test 
+the day of or the night before boarding the train to Finse. If you test positive, you 
+should not travel to Finse. We also request testing the following couple of days.
+- Everybody brings sufficient masks (FFP2/N95 or similar, no cloth masks). Wearing masks 
+will be optional first, but will become necessary if someone tests positive. 
+- If anybody tests positive for covid, the person will need to isolate and can no longer 
+participate in the workshop in person. We may adjust the testing protocol as appropriate.
+
+We hope that you agree with this protocol, which aims at everyone’s safety. 
+Please get in touch with us if you have any questions!
+
+### Registration
+If you wish to participate in the workshop, please fill out the following 
+[form, WE NEED A NEW FORM AND LINK]().
+
+Due to the limited capacity in accommodation, **we would kindly like to ask you to do so 
+before 31 May** (indicating your interest does not mean you're registered right away for the workshop). 
```

Comment:
Maybe we can say before April 30th? So we can invite people to register in the first half of May, after that it would be than be first come first serve.